crates/bevy_mikktspace/examples/generate.rs

```diff
@@ -0,0 +1,259 @@
+#![allow(clippy::bool_assert_comparison, clippy::useless_conversion)]
```

Comment:
The CI target is failing so we will need to decide whether we suppress the clippy warnings for the entire file, or go through and fix them all. I would rather not fix them in this initial commit because I don't want to touch the code any more than necessary. Any changes we make would then be bisectable back to this, which is better for debugging.

Reply:
I'd prefer to suppress these first and then follow up later with fixes if we see the need. That way we have the original state in our history.

Reply:
Just suppressed the remaining lints (we recently merged some new ones).